helpers/components/addInArticleAds.test.js

```diff
@@ -0,0 +1,211 @@
+    },
+    subsection = {
+      _ref: 'localhost/components/subsection/instances/fixture',
+      content: [
+        paragraph4, paragraph1
+      ]
+    },
+    nonTextComponent = {
+      _ref: 'localhost/components/other-component/'
+    },
+    mediaplayComponent = {
+      _ref: 'localhost/components/mediaplay/'
+    },
+    relatedStoriesComponent = {
+      _ref: 'localhost/components/related-stories/'
+    },
+    ooyalaPlayerComponent = {
+      _ref: 'localhost/components/ooyala-player/',
+      videoId: 'lj09283r098dfg'
+    },
+    desktopOutStreamAd = {
+      _ref: 'localhost/components/ad/desktopOutStream/'
+    },
+    desktopPremiumAd = {
+      _ref: 'localhost/components/ad/desktopPremium/'
+    },
+    tabletAd = {
+      _ref: 'localhost/components/ad/tablet/'
+    },
+    firstMobileAd = {
+      _ref: 'localhost/components/ad/mobileFirst/'
+    },
+    otherMobileAd = {
+      _ref: 'localhost/components/ad/mobileOther/'
+    },
+    featureTypesCoverStory = {
+      'Cover Story Online': true
+    },
+    featureTypesRegularArticle = {
+      'Cover Story Online' : false
+    },
+    readMore = {
+      _ref: 'localhost/components/read-more/instances/long',
+      content: [paragraph1, paragraph4, paragraph1, paragraph4, paragraph4]
+    };
+
+  it('passes through content of an article', function () {
+    expect(filter(content)).to.deep.equal(content);
```

Comment:
not sure what the best way is to test with actual handlebars templates (e.g., using `const tpl = '{{ . . . `)?